packages/frontend/amp/components/Body.tsx

```diff
@@ -8,11 +8,19 @@ import { TopMetaOpinion } from '@frontend/amp/components/TopMetaOpinion';
 import { SubMeta } from '@frontend/amp/components/SubMeta';
 import { pillarPalette } from '../../lib/pillars';
 import { palette } from '@guardian/pasteup/palette';
+import { getToneType, Tone } from '../lib/tag-utils';
 
-const body = (pillar: Pillar, isOpinion: boolean) => css`
-    background-color: ${isOpinion ? palette.opinion.faded : 'white'};
-    ${bulletStyle(pillar)}
-`;
+const body = (pillar: Pillar, tone: Tone) => {
+    const bgColorMap = {
+        isDefault: 'white',
```

Comment:
It is possible to use `palette.neutral[100]` for this? I think when this was first written we didn't have the palette system, but I should have updated it when I introduced the isOpinion malarkey! Thanks